content_schemas/formats/historic_appointment.jsonnet

```diff
@@ -1,9 +1,48 @@
 (import "shared/default_format.jsonnet") + {
+  document_type: [
+    "historic_appointment"
+  ],
   definitions: {
     details: {
       type: "object",
       additionalProperties: false,
-      properties: {},
+      required: [
+        "body",
+        "political_party",
```

Comment:
Question: are we sure these are the only required fields? Are there examples of historic appointment pages without e.g. birth dates? 

(I'm sure you've got good reason for this, just checking)

Reply:
I took these from Whitehall's validations performed by Whitehall on the editing form for this content (e.g. https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/people/tony-blair/historical_accounts/2/edit).  It feels like 'born' should be required, but I didn't want a mismatch with the publishing app.

Political parties aren't marked as compulsory in Whitehall's UI, but it gives an error if not included.